utils/utils.c

```diff
@@ -44,7 +44,7 @@ glusterBlockSetLogLevel(unsigned int logLevel)
   LOCK(gbConf.lock);
   gbConf.logLevel = logLevel;
   UNLOCK(gbConf.lock);
-  LOG("mgmt", GB_LOG_INFO,
+  LOG("mgmt", GB_LOG_NONE,
```

Comment:
@lxbsz Looks like you are fixing this part in https://github.com/gluster/gluster-block/pull/163/commits/94d6d9ab52edc050efe5a41700f53fb26ed229c8 with GB_LOG_CRIT already. Can we close this PR ?

Reply:
Yeah, right.
Since this is duplicated to @163, I will close this PR now.
Thanks Prasanna.